### ISSUE TITLE
new error message received for reader upload "unexpected issue with database.  no changes made."

### ISSUE THREAD

**Issue by baeolophus** [etag/etagq#7]
uploaded same test file twice and got message "Unexpected issue with database. No changes made."   In Firefox.  API link: https://head.ouetag.org/api/queue/task/159abeb5-1c5f-46dd-80b3-31a82c6ff8ba/

**Comment by baeolophus** [comment]
Nope, not code logic update I think, as tried again with different files and it worked with the updated readers = 3 message I expected.  Possibly not a problem, although when I try to upload a second file without refreshing the page it hangs.

**Comment by baeolophus** [comment]
also reported by workshop2 attendee with same message, sample dataset 2, for reader data.

**Comment by baeolophus** [comment]
I think it turned out to be duplicate reader_ids?  so improved error messages here.

**Comment by tdpearson** [comment]
The underlying cause for a majority of these was due to input data exceeding the maximum field length. Improved handling and messaging to the user of exceeding maximum field lengths is a good direction.
